ios/FluentUI/DrawerVnext/Drawer+Drag.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+internal extension Drawer {
+    func dragGesture(screenWidth: CGFloat) -> some Gesture {
+        DragGesture()
+            .onChanged { value in
+                let withinDragBounds = state.presentationDirection == .left ? value.translation.width < 0 : value.translation.width > 0
+                if withinDragBounds {
+                    draggedOffsetWidth = value.translation.width
+                }
+            }
+            .onEnded { _ in
+                if let draggedOffsetWidth = draggedOffsetWidth {
+                    if abs(draggedOffsetWidth) < screenWidth * gestureSnapWidthRatio() {
+                        state.isExpanded = true
+                    } else {
+                        state.isExpanded = false
+                    }
+                }
+            }
+    }
+
+    private func gestureSnapWidthRatio() -> CGFloat {
+        return 0.225
```

Comment:
Should this be tokenized?

Reply:
@sophialee0416 tokens are only applied to appearances and not behavioral attributes

@rdeassis  we can't add const static property in Drawer extension because of it's generic type. 
I am also not a big fan of redundant documentation. 

this is from FluentUI [drawer](https://github.com/microsoft/fluentui-apple/blob/bb4d1ac255b858755f25ea427ad2305a66c90b0f/ios/FluentUI/Drawer/DrawerController.swift#L262) 
```swift
// If the drawer is already loaded, update the resizing handle ...
 if isViewLoaded {
       updateResizingHandleView()
}
```

Redundant documentation creates a lot of problems than in solves. documentation also goes out of sync more often that code does.

I am like to add comments when they are [necessary](https://blog.cleancoder.com/uncle-bob/2017/02/23/NecessaryComments.html). 

I intentionally create small method where it's name eliminates the need for documentation and I believe `gestureSnapWidthRatio` in Drag extension has sufficient context to interpret the meaning of the value. I am open to other suggestions to improve nomenclature or restructure code to eliminate the need of documentation.